app/subscriber.py

```diff
@@ -43,7 +43,9 @@ def receipt_to_case(message: Message):
     try:
         payload = json.loads(message.data)  # parse metadata as JSON payload
         metadata = payload['metadata']
-        case_id, tx_id = metadata['case_id'], metadata['tx_id']
+        tx_id, questionnaire_id, case_id = metadata['tx_id'], metadata['questionnaire_id'], 0
```

Comment:
```suggestion
        tx_id, questionnaire_id, case_id = metadata['tx_id'], metadata['questionnaire_id'], metadata.get('case_id')
```
To default it to `None` if it is missing.

Reply:
ah thats nice 👍